modules/EmailMan/language/en_us.lang.php

```diff
@@ -132,9 +132,11 @@
     'LBL_PREPEND_TEST' => '[Test]: ',
     'LBL_SEND_ATTEMPTS' => 'Send Attempts',
     'LBL_OUTGOING_SECTION_HELP' => 'Configure the default outgoing mail server for sending email notifications, including workflow alerts.',
-    'LBL_ALLOW_DEFAULT_SELECTION' => 'Allow users to use this account for outgoing email:',
-    'LBL_ALLOW_DEFAULT_SELECTION_HELP' => 'When this option selected, all users will be able to send emails using the same outgoing<br> mail account used to send system notifications and alerts. If the option is not selected,<br> users can still use the outgoing mail server after providing their own account information.',
+    'LBL_ALLOW_DEFAULT_SELECTION' => 'Users may send as this account:',
```

Comment:
Can the wording be improved? I'm not an english user here but it seems it would be difficult to translate in a meaningful way

Reply:
@horus68 
I don't speak any language other than (US) English, so I'm not sure what the trouble may be.
Perhaps 'send' is ambiguous without context?
It does need to be shortened so that it doesn't mess with the layout. Maybe there is a better way to avoid the layout issue?

Reply:
I suggest "users may send with this account's identity"

Reply:
Changed!

Reply:
@horus68 @pgorod 
Changed!